tests/dlq_test.go

```diff
@@ -524,3 +646,11 @@ func (t testExecutor) Execute(ctx context.Context, e queues.Executable) queues.E
 	}
 	return t.base.Execute(ctx, e)
 }
+
+// ReadTasks is used to block the dlq job workflow until one of them is cancelled in TestCancelRunningMerge.
+func (m *testTaskQueueManager) DeleteTasks(ctx context.Context, request *persistence.DeleteTasksRequest) (*persistence.DeleteTasksResponse, error) {
+	select {
+	case <-m.suite.deleteBlockCh:
+		return m.HistoryTaskQueueManager.DeleteTasks(ctx, request)
+	}
```

Comment:
I don't think there's a point in having a select since we just have one case

Reply:
Right. Made the change.